remoting/zookeeper/listener.go

```diff
@@ -129,14 +129,14 @@ func (l *ZkEventListener) handleZkNodeEvent(zkPath string, children []string, li
 			continue
 		}
 		// listen l service node
-		go func(node string) {
+		go func(node, childNode string) {
```

Comment:
`zkPath` may also add to parameter?

Reply:
@AlexStocks why add childNode ,just use in log?